### PR TITLE
Constant Multiple Feature branch

### PR DESCRIPTION
**Pull request by johnnieskywalker**
# [Create Constant Multiple Form](https://app.shortcut.com/oazo-apps/story/5216/create-constant-multiple-form)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- item 1
  
## How to test 🧪
  <Please explain how to test your changes>
- step 1 ...

### CODE REVIEW

**Comment by johnnieskywalker** [comment]
image

**Comment by johnnieskywalker** [comment]
image

**Review by piekczyk — CHANGES_REQUESTED** [review]

File: components/vault/detailsSection/ContentCardTargetMultiple.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+export function ContentCardTargetMultiple({
+  targetMultiple,
+  targetColRatio,
+  afterTargetMultiple,
+  changeVariant,
+}: ContentCardTargetMultipleProps) {
+  const { t } = useTranslation()
+
+  const formatted = {
+    targetMultiple: `${targetMultiple?.toFixed(1)}x`,
```
[the remark this answers is not in the document]
Comment (piotrkonowrocki) [reply components/vault/detailsSection/ContentCardTargetMultiple.tsx]: Fixed.

File: features/automation/optimization/controls/ConstantMultipleDetailsLayout.tsx [hunk 2]
```diff
@@ -1,66 +1,92 @@
...
+              <ContentCardTargetMultiple
+                targetMultiple={targetMultiple}
+                afterTargetMultiple={uiState.multiplier}
+                targetColRatio={targetColRatio}
+                changeVariant="positive"
+              />
+              <ContentCardTotalCostOfFeature
+                totalCost={totalCost}
+                // TODO: PK: how to calculate this value change dynamically?
+                afterTotalCost={undefined}
```
[the remark this answers is not in the document]
Comment (piotrkonowrocki) [reply features/automation/optimization/controls/ConstantMultipleDetailsLayout.tsx]: Fixed.

File: features/automation/optimization/sidebars/SidebarSetupConstantMultiple.tsx [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+    stopLossTriggerData,
+    ilkData,
+  })
+  const sliderMax = min([
+    vault.lockedCollateralUSD
+      .div(ilkData.debtFloor)
+      .multipliedBy(100)
+      .decimalPlaces(0, BigNumber.ROUND_DOWN)
+      .toNumber(),
+    SLIDER_MAX_FOR_BIG_VAULTS,
```
[the remark this answers is not in the document]
Comment (johnnieskywalker) [reply features/automation/optimization/sidebars/SidebarSetupConstantMultiple.tsx:117]: we discussed it with @adamskrodzki see comments in task https://app.shortcut.com/oazo-apps/story/5216/create-constant-multiple-form

**Review by piekczyk — DISMISSED** [review]

**Review by piotrkonowrocki — APPROVED** [review]

**Review by piekczyk — APPROVED** [review]

**Pull request merged**